lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
```

Comment:
We could probably just pull the confidence computation into this helper function, then.